### PR TITLE
[Feature][Monitoring] Added disk available capacity monitoring

### LINKED ISSUE #10190: [Bug] [E2E] The front-end page changes failed the E2E test
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

relevance   https://github.com/apache/dolphinscheduler/pull/10186

https://github.com/apache/dolphinscheduler/runs/6548541522?check_suite_focus=true
image

### What you expected to happen

pass

### How to reproduce

pass

### Anything else

_No response_

### Version

3.0.0-alpha

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by retime123**
## Purpose of the pull request
This method is added because the available disk capacity cannot be monitored  

## Brief change log

Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java
Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java
Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java
Change dolphinscheduler-ui/src/views/monitor/servers/master/index.tsx
Change dolphinscheduler-ui/src/views/monitor/servers/worker/index.tsx
Change dolphinscheduler-ui/src/locales/modules/en_US.ts
Change dolphinscheduler-ui/src/locales/modules/zh_CN.ts

unit testing
Change dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java
Change dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/HeartBeatTest.java

## Verify this pull request

retry

image

### CODE REVIEW

**Comment by songjianet** [comment]
Please create an issue and associate with it. @retime123

**Comment by retime123** [comment]
image
E2E,Still can't pass!  I need you guys to take a look

**Comment by SbloodyS** [comment]
@retime123 You can take a look at this error.

**Comment by retime123** [comment]
> 
> 
> @retime123你可以看看这个错误。

Can't see the specific error!  Can you send me the generated logs?  Or could you show me how to change it

**Comment by SbloodyS** [comment]
> > 
> > @retime123你可以看看这个错误。
> 
> Can't see the specific error! Can you send me the generated logs? Or could you show me how to change it

The error logs in ```https://github.com/apache/dolphinscheduler/runs/6548541522?check_suite_focus=true```

**Comment by retime123** [comment]
Can this function be combined?

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java [hunk 1]
```diff
@@ -92,6 +93,24 @@ public static double memoryUsage() {
...
 
+    /**
+     * get disk usage
+     * Keep 2 decimal
+     *
+     * @return disk free space  (GB)
+     */
+    public static double diskAvailable() {
+        File file = new File(".");
+        long totalSpace = file.getTotalSpace(); //total disk space in bytes.
```
Comment (caishunfeng) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java]: It seems not used.

**Comment by caishunfeng** [comment]
Hi @retime123 I found that you had submit a same pr  and the comment which not resolve https://github.com/apache/dolphinscheduler/pull/10041#pullrequestreview-973619974, 
can you test in k8s?

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java (hunk 1 above)
Comment (retime123) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java]: totalSpace     Yes. It's not used

Comment (caishunfeng) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java]: > totalSpace Yes. It's not used

Please remove it if not use.

Comment (retime123) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java]: Resubmitted

**Comment by retime123** [comment]
> Hi @retime123 I found that you had submit a same pr and the comment which not resolve [#10041 (review)](https://github.com/apache/dolphinscheduler/pull/10041#pullrequestreview-973619974), can you test in k8s?

This probably has nothing to do with K8S.

**Comment by retime123** [comment]
image
It's always a surprise！
Run try again

**Comment by caishunfeng** [comment]
> image It's always a surprise！ Run try again

I rerun it.

**Review by caishunfeng — APPROVED** [review]
LGTM of the backend part.  PTAL @songjianet @labbomb

**Comment by caishunfeng** [comment]
@retime123 please resolve the conflicts.

**Comment by retime123** [comment]
> 

This is refactoring! I'll resubmit it

**Pull request closed without merging**